src/main/java/bean/Basket.java

```diff
@@ -1,11 +1,73 @@
 package bean;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "Basket")
 public class Basket {
+    @Id
+    @Column(name = "id")
     int id;
+    @Column(name = "book_id")
     int bookId;
```

Comment:
If this is Hibernate entity, then it SHould be the corresponding Object instead of promitive
Eg.
@ManyToOne
@JoinColumn((name = "book_id"))
private Book book;

In other appropriate places should be the same